cmd/promote/git/git_cmd.go

```diff
@@ -22,50 +22,8 @@ func getBaseDir(exec iexec.IExec) (string, error) {
 			return
 		}
 
-		BaseDir = strings.TrimSpace(string(baseDirOutput))
+		BaseDir = strings.TrimSpace(baseDirOutput)
 	})
 
 	return BaseDir, baseDirErr
 }
-
-/*
-// Not in use
```

Comment:
hm

Reply:
Unsure why this was commented out as "not in use" versus just being removed.